gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -1902,6 +1913,17 @@ _gtfc_setprop_SHARED_TENSOR_FILTER_KEY (GstTensorFilterProperties * prop,
   return 0;
 }
 
+/** @brief Handle "PROP_INVOKE_DYNAMIC" for set-property */
```

Comment:
How about ?
/** 
  * @brief Handle "PROP_INVOKE_DYNAMIC" for set-property 
  */

Reply:
Updated. Thanks.